.github/workflows/release-runners.yaml

```diff
@@ -3,23 +3,19 @@ name: Runners
 # Revert to https://github.com/actions-runner-controller/releases#releases
 # for details on why we use this approach
 on:
-  # We must do a trigger on a push: instead of a types: closed so GitHub Secrets 
+  # We must do a trigger on a push: instead of a types: closed so GitHub Secrets
   # are available to the workflow run
   push:
     branches:
       - 'master'
     paths:
-      - 'runner/**'
-      - '!runner/Makefile'
-      - '.github/workflows/runners.yaml'
-      - '!**.md'
+      - 'runner/VERSION'
```

Comment:
This doesn't factor in the changes to the container-hooks version or the docker version

Reply:
We weren't before either? Adding this workflow file to `paths`.

Reply:
Unless you meant updating those automatically too, but discussing approaches yesterday it seems we're very intentional with releasing new container hooks versions and they're not necessarily tied to runner releases.

Reply:
Nevermind this comment, I didn't see this line 🤦‍♂️ we're all good here

`- '.github/workflows/release-runners.yaml'`